pkg/datapath/linux/node.go

```diff
@@ -897,6 +899,25 @@ func (n *linuxNodeHandler) nodeUpdate(oldNode, newNode *nodeTypes.Node, firstAdd
 		oldKey = oldNode.EncryptionKey
 	}
 
+	if !n.nodeConfig.EnableIPSec {
+		bpfIPCacheList := make(map[string][]string)
+		if err := ipcache.IPCache.Dump(bpfIPCacheList); err != nil {
```

Comment:
I'm checking for the key in ipcache in the function `nodeUpdate`, is this the correct function to check? or should I shift this to `NodeConfigurationChanged()`